pkg/process/config/config_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
```

Comment:
Mind splitting into a separate import section?

Reply:
Oh thanks, I missed that one